ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/dropdownActivityPicklistQuestion.component.ts

```diff
@@ -17,7 +17,7 @@ import { NGXTranslateService } from '../../../services/internationalization/ngxT
                         [placeholder]="block.picklistLabel"
                         [disabled]="readonly"
                         [multiple]="true"
-                        (selectionChange)="handleMaterialSelect($event)"
+                        (selectionChange)="handleMaterialSelect($event); details.show ? updateCharactersLeftIndicator(details.stableId) : null"
                         class="width">
```

Comment:
Similar refactoring would be helpful here too.

Reply:
I wanted to pass options object in the `updateCharactersLeftIndicator(...)` instead of object.stableId, but I want to reject this refactoring because `CheckboxesActivityPicklistQuestion` component and `RadioButtonsActivityPicklistQuestion` component use `ActivityPicklistOption` model for option, but `DropdownActivityPicklistQuestion` component doesn't use info from option, it uses `ActivityPicklistDetails` for details